tasks/install-mdex.yml

```diff
@@ -75,6 +87,15 @@
 
 - name: "Become section"
   block:
+
+    - name: "Enable connections to {{ mdex_port }} port"
+      seport:
+        ports: "{{ mdex_port }}"
+        proto: "tcp"
+        setype: "http_port_t"
+        state: "present"
+      when: ansible_selinux.status == "enabled" and ansible_selinux.mode != "disabled"
+
```

Comment:
'when' conditionals can be a list in that case:
```yaml
when: 
  - ansible_selinux.status == "enabled"
  - ansible_selinux.mode != "disabled"
```

Reply:
Does it make sense for 2 conditions?

Reply:
As discussed

Reply:
Fixed